tool/travis.sh

```diff
@@ -48,10 +48,7 @@ else
   echo "Running main linter bot"
 
   # Verify that the libraries are error free.
-  dartanalyzer --fatal-warnings \
-    bin/linter.dart \
-    lib/src/rules.dart \
-    test/all.dart
+  dart analyze --fatal-warnings --fatal-infos .
```

Comment:
`--fatal-warnings` defaults to true; consider just using `--fatal-infos`

Reply:
Good catch.